app/src/main/java/com/example/weatherapp/models/WeatherMainMenuViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.weatherapp.models
+
+import kotlinx.coroutines.flow.MutableStateFlow
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+
+class WeatherMainMenuViewModel : ViewModel() {
```

Comment:
I think we can rename this viewmodel to `MainViewModel` or `LocationListViewModel`